### ISSUE TITLE
Failed training on GREW when trained with more than one GPU

### ISSUE THREAD

**Issue by 1292224662** [ShiqiYu/OpenGait#131]
What I saw: I used the defaulf gaitbase_da_grew.yaml and dataset to train on GREW. However, when I used 4 GPUs or 2 GPUs, the loss quickly (no more than 5k iters) became nan, while it didn't happen when I used only 1 GPU.

Question: I guess it has something to do with learning rate. Is the optimizer_cfg::lr in yaml only the learning rate on 1 GPU, so when I used 4 GPUs, I should changed it to "optimizer_cfg::lr = 0.025" if I want the real learning rate to be 0.1 ?
